packages/frontend/src/renderer/components/widgets/channelSettings/Notifications.tsx

```diff
@@ -109,91 +99,33 @@ export const Notifications: React.FC<NotificationsProps> = ({
-                  }
-                  onChange={() =>
-                    setChannelsNotification(notificationFilterType.MENTIONS)
-                  }
-                  label='Just @mentions'
-                />
-                <FormControlLabel
-                  classes={{ root: classes.radioIcon }}
-                  control={
-                    <Checkbox
-                      icon={<Icon src={radioUnselected} />}
-                      checkedIcon={<Icon src={radioChecked} />}
-                      checked={notificationFilterType.NONE === currentFilter}
-                    />
-                  }
-                  onChange={() =>
-                    setChannelsNotification(notificationFilterType.NONE)
-                  }
-                  label='Nothing'
-                />
+
+            <Grid item container direction='column'>
+              <Grid item>
+                <Typography variant='h4'>
+                  You've muted this channel
+                </Typography>
               </Grid>
-            ) : (
-              <Grid item container direction='column'>
-                <Grid item>
-                  <Typography variant='h4'>
-                    You've muted this channel
-                  </Typography>
-                </Grid>
-                <Grid item className={classes.infoDiv}>
-                  <Typography variant='body2'>
-                    Unmute this channel to change your notification settings.
-                  </Typography>
-                </Grid>
-                <Grid>
-                  <Button
-                    variant='contained'
-                    size='large'
-                    color='primary'
-                    type='submit'
-                    fullWidth
-                    className={classes.button}
-                    onClick={() => {
-                      setChannelsNotification(
-                        notificationFilterType.ALL_MESSAGES
-                      )
-                    }}
-                  >
-                    Unmute Channel
-                  </Button>
-                </Grid>
+              <Grid item className={classes.infoDiv}>
+                <Typography variant='body2'>
+                  Unmute this channel to change your notification settings.
+                </Typography>
               </Grid>
-            )}
+              <Grid>
+                <Button
+                  variant='contained'
+                  size='large'
+                  color='primary'
+                  type='submit'
+                  fullWidth
+                  className={classes.button}
+                  onClick={() => { }}
+                >
+                  Unmute Channel
+                </Button>
+              </Grid>
+            </Grid>
+
             <Grid item className={classes.captionDiv}>
               <Typography variant='caption' className={classes.captions}>
                 You can choose how to be alerted or turn off all Quiet
```

Comment:
there are unused files, we have 'settings' folder in widgets now instead of that 'channelSettings', Bartek removed this files in his bump electron pr, i just changed it in components and containers when i removed old store structure and replaced with new 'settings' store structure in nectar, but finally its unused and will be removed